Add the importance of POL (#100)
OlympusDAO-Education/Documentation: basics/basics.md

## How does it work?

At a high level, OlympusDAO consists of its protocol managed treasury, protocol

> owned liquidity \([POL](../references/glossary.md#POL)\), bond mechanism, and high

Reviewer: Maybe as an idea let's not emphasize "high" this much anymore. We will gradually adjust the reward rate from here. 

```suggestion
owned liquidity \([POL](../references/glossary.md#POL)\), bond mechanism, and
```

staking rewards that are designed to control supply expansion.

> Bond generates profit for the protocol, and the treasury uses the profit to mint

Reviewer: ```suggestion
Bond sales generate profit for the protocol, and the treasury uses the profit to mint
```

As the protocol accumulates more PCV, more runway is guaranteed for the stakers. This means the stakers can be confident that the current staking APY can be sustained for a longer term because more funds are available in the treasury.

## Why is POL important?

Olympus [owns most of its liquidity](https://dune.xyz/shadow/Olympus-\(OHM\)) thanks
to its bond mechanism. This has several benefits:

> - Olympus does not have to pay out high farming rewards to incentivize liquidity.

Reviewer: Zeus often refers to it like that. 

```suggestion
- Olympus does not have to pay out high farming rewards to rent liquidity.
```

Author: See my comment below.

> - Olympus guarantees the market that the liquidity is always there to facilitate

Reviewer: Fine with that.

sell or buy transaction.
- By being the largest LP (liquidity provider), it earns most of the LP fees which

> represents another source of income to the treasury.

Reviewer: Maybe something like this as well. 

```suggestion
represents another source of income to the treasury.
- All POL represents productive assets marked down to back OHM. 
```

Author: See my comment below.

> represents another source of income to the treasury.

Author: Elaborate on the marking down of LP tokens.

> represents another source of income to the treasury.

Reviewer: lessgo
